src/simularium/VisDataCache.ts

```diff
@@ -1,4 +1,4 @@
-import { compareTimes } from "../util";
+import { compareTimes } from "../util.js";
 import { CachedFrame, CacheNode } from "./types";
```

Comment:
do we also need a .js after types?

Reply:
You would think so, it seems to be working fine with or without and adding it would be more consistent.